src/pageObject.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the Conduit logo';
+  };
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
+    return '[Publish comment] button';
+  };
+  clickOnCommentButton() {
+    return 'Click on the [Publish comment] button';
```

Comment:
should contain clickOnCommentButton method. The method should return a string 'Click on the' + commentButton

you need to use commentButton property here
```suggestion
    return 'Click on the [Publish comment] button';
```